services/common/src/utils/featureFlag.ts

```diff
@@ -12,6 +12,7 @@ export enum Feature {
   VC_ANONCREDS_MINESPACE = "vc_anoncreds_minespace",
   VC_W3C = "vc_w3c",
   MINESPACE_ESUPS = "minespace_esups",
+  MINESPACE_NOW_STATUS = "minespace_now_status",
```

Comment:
Just a reminder to make sure this gets added to the live environments.  I don't think I see it there yet.